openapi.yaml

```diff
@@ -17305,6 +17459,68 @@ components:
           x-linode-cli-color:
             None: black
             default_: white
+    LinodeConfigInterface:
+      type: object
+      description: >
+        The Network Interface to apply to this Linode's configuration profile.
+      properties:
+        label:
+          type: string
+          minLength: 0
+          maxLength: 64
+          pattern: '/[a-z0-9-]+/'
+          description: |
+            The name of this interface.
+
+            Only used for type `vlan` interfaces. Enter an empty string for type `public` interfaces.
+
+            Interface labels must be unique within the data center region.
+          example: example-interface
+        ipam_address:
+          type: string
+          format: ip/netmask
+          description: |
+            This Network Interface's private IP address in Classless Inter-Domain Routing (CIDR) notation.
+
+            Only used for type `vlan` interfaces. Enter an empty string for type `public` interfaces.
```

Comment:
Can we note here that an empty string is allowed for the 'vlan' interfaces also? The customer would have to manually configure the network in this case.